webservices/docs.py

```diff
@@ -2098,7 +2098,7 @@ def add_ytd(var):
 '''
 
 HITS_RETURNED = '''
-Number of results to return (max 10)
+Number of results to return. The default value is 20, with a maximum limit of 200 results per page
```

Comment:
@pkfec If you are changing rulemakings default to 30 (thank you!), will the message on swagger docs for rulemaking will be wrong, because it uses the same string as legal docs?

Reply:
@johnnyporkchops I will go ahead a create a new string for rulemaking